### DISCUSSION TITLE
Custom Villager trade  in CMI ?

### DISCUSSION THREAD

**Discussion by Blueeyestar** [Zrips/CMI#1683] [Enhancement]
How do you do?
is it possible that you can do it so that you can set the prices at the villagers via cmi?
We don't want the villagers to sell mending because they are op.
We want to set the prices ourselves so that the villagers do not ruin the trade on the server. Do you have a possibility to do something with CMI in the 1.14? 
Other plugins that could do that are broken since the 1.14 like the plugin Custom Village Trades 0.11.0.

**Comment by JohOply** [comment]
I agree with this idea
It could be useful to be able to custom every trades, and change vanilla trades :)
